docs/notebooks/tutorials/EffectiveRipple.ipynb

```diff
@@ -0,0 +1,480 @@
+{
```

Comment:
<p>may want to reduce the resolution of the equilibrium itself to make this a bit faster. for the ballooning one I did (L,M,N)=(12,6,2), grids=(18,12,4)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaControl/DESC/pull/1290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
better done on #1425